### PR TITLE
Add awk bats tests

### PR DESCRIPTION
**Pull request by giusdp**
This PR adds some awk.bats tests

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -127,7 +127,7 @@ func main() {
 		if tools.IsTool(cmd) {
 			code, err := tools.RunTool(cmd, args[2:])
 			if err != nil {
-				log.Print(err.Error())
+				fmt.Println(err.Error())
```
Comment (msciabarra) [inline main.go:130]: why this change? log will send in stdout, fmt will send in stderr - I do not want errors in stdout!

Comment (giusdp) [reply main.go:130]: Oops sorry I forgot about it. The problem I was having is that log prints date and time in front of the message. With the forst awk test the `assert_line` it was failing because it was never matching, i changed it just to go on with the tests and forgot
